### PR TITLE
SAK-42423 Google Drive user table

### PR DESCRIPTION
**Pull request by bgarciaentornos**

### CODE REVIEW

File: docs/conversion/sakai_20_mysql_conversion.sql [hunk 1]
```diff
@@ -70,3 +70,15 @@ DROP TABLE PERMISSIONS_SRC_TEMP;
 ALTER TABLE SAM_ITEM_T ADD ISEXTRACREDIT TINYINT(1);
 ALTER TABLE SAM_PUBLISHEDITEM_T ADD ISEXTRACREDIT TINYINT(1);
 -- END SAK-34741
+
+-- SAK-42423 GOOGLEDRIVE
+CREATE TABLE GOOGLEDRIVE_USER (
+  sakaiUserId varchar(255) NOT NULL,
```
Comment (ottenhoff) [inline docs/conversion/sakai_20_mysql_conversion.sql]: The Sakai user ID should be 99 chars

File: docs/conversion/sakai_20_mysql_conversion.sql [hunk 2]
```diff
@@ -70,3 +70,15 @@ DROP TABLE PERMISSIONS_SRC_TEMP;
...
+  googleDriveName varchar(255) DEFAULT NULL,
+  refreshToken longtext,
+  googleDriveUserId varchar(255) DEFAULT NULL,
+  token longtext,
```
Comment (ottenhoff) [inline docs/conversion/sakai_20_mysql_conversion.sql]: How large can the token be?

Comment (bgarciaentornos) [reply docs/conversion/sakai_20_mysql_conversion.sql]: It's variable but I've seen some tokens taking up to 1200 chars

Comment (ottenhoff) [reply docs/conversion/sakai_20_mysql_conversion.sql]: Got it. Thanks

**Pull request merged**